softlayer/softlayer_virtual_guest_service.go

```diff
@@ -40,4 +40,6 @@ type SoftLayer_Virtual_Guest_Service interface {
 
 	AttachIscsiVolume(instanceId int, volumeId int) (string, error)
 	DetachIscsiVolume(instanceId int, volumeId int) error
```

Comment:
We can add detachEphemeralDisk for softlayer-go, can submit this in another independent story. But I'm not seeing if BOSH needs this feature till now?